### PR TITLE
Card slider updates

### PR DESCRIPTION
**Pull request by richarddubay**
# Feature / Fixed Issue(s)
- The card slider now takes components with data already baked in and renders them.

# Testing/Documentation
- [x] All significant new logic is covered by tests.
- [x] Storybook documentation has been added.

### CODE REVIEW

**Comment by jbaxleyiii** [comment]
**[Storybook Hub](https://hub.getstorybook.io/)** is building storybooks for every commit in this PR.

**Latest Storybook: https://36f4485a-2e6f-4bfb-b996-0e38b150e6c0.sbook.io/**
<sub>( Visit [here](https://hub.getstorybook.io/app/github.com/NewSpring/Holtzman/branch/card-slider-updates/) for more info )</sub>

> _This comment is automatically added by Storybook Hub_

File: imports/components/@primitives/UI/card-slider/SliderCard.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
 // @flow
 
 type ISliderCard = {
```
Comment (jbaxleyiii) [inline imports/components/@primitives/UI/card-slider/SliderCard.js]: @richarddubay I should have caught this earlier on. This isn't really a generic enough slider card. Can we rename it to MetricCard?

File: imports/components/@primitives/UI/card-slider/index.js [hunk 2]
```diff
@@ -90,17 +66,17 @@ const CardSlider = ({
         WebkitOverflowScrolling: "touch",
       }}
     >
-      <section style={dynamicWidthContainer(cardData.length)}>
-        {cardData.map((x, key) => {
+      <section style={dynamicWidthContainer(childCount)}>
+        {children.map((component, key) => {
```
Comment (jbaxleyiii) [inline imports/components/@primitives/UI/card-slider/index.js]: super cool 👍

File: imports/components/@primitives/UI/card-slider/SliderCard.js (hunk 1 above)
Comment (richarddubay) [inline imports/components/@primitives/UI/card-slider/SliderCard.js]: Do you want me to move it under cards as well?

**Pull request merged**